Monika After Story/game/script-topics.rpy

```diff
@@ -4744,6 +4744,44 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['donation', 'donations', 'fan pack', 'fanpack']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_write')
+    monika_random_topics.append('monika_write')
```

Comment:
The topic you are adding here is incorrect. Please change this (and the line above it) to the topic you are adding.